submissions/DaniaB24/js/app.js

```diff
@@ -0,0 +1,120 @@
+const cardsOfFriends = document.getElementById("allHumans");
+const searchBar = document.getElementById("search");
+
+let filteredFriends;
+let allFriends;
+let sortedFriends;
+let friends;
+fetch("https://randomuser.me/api/?results=10")
+  .then((response) => response.json())
```

Comment:
I would suggest a shallow copy. Because if you will mutate array - it will be also mutated by link.

Reply:
thanks for the review, I will do it🫡